packages/run/docker-compose.yml

```diff
@@ -0,0 +1,142 @@
+# Environment variable definitions
```

Comment:
can we actually run `nwaku-compose` with needed configs for us here?

Reply:
nwaku-compose comes with a bunch of other stuff we don't need (e.g. grafana/prometheus)